js/ui/codeView.js

```diff
@@ -453,6 +457,63 @@ var CodingSession = new Lang.Class({
         return this._toolbox;
     },
 
+    setGrabbed: function(grabbed) {
+        if (this._grabTimeoutId > 0) {
+            Mainloop.source_remove(this._grabTimeoutId);
+            this._grabTimeoutId = 0;
+        }
+
+        if (!this._button)
+            return;
+
+        if (grabbed)
+            this._button.opacity = 0;
+        else {
```

Comment:
I'm not sure what's the convention here (I had the impression that as long as you have a block with curly brackets, the other should have them too). So let's wait for someone else to clarify that.

Reply:
I think you're right about this convention, I've configured the eslint for the shell and it tells me that I should put brackets there. I've updated it.